### ISSUE TITLE
Can I replace 填充 to prollyfill on ch5 of async & performance, branch 1ed-zh-CN?

### ISSUE THREAD

**Issue by IronBlood** [getify/You-Dont-Know-JS#1761]
Hi, Kyle, thank you for you books! I already searched for this issue as nobody mentioned "prollyfill" in issues.

**Edition:** 1st

**Book Title:** async & performance

**Chapter:** 5

**Section Title:** SIMD

**Problem:**

I read the paperback version [ISBN 9787115431165](https://www.amazon.com/dp/7115431167) and found a typo (or maybe not?), on the note they use *prolyfill* (shown as the picture below) instead of [*prollyfill*](https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/async%20%26%20performance/ch5.md#simd). It might be a little bit off-the-topic but I wonder 1) future Chinese translators might take a glance at issues or 2) other Chinese readers may check and read the 1ed-zh-CN branch, so I made a [patch](https://github.com/getify/You-Dont-Know-JS/compare/1ed-zh-CN...IronBlood:1ed-zh-CN), and here's my thought.

The community-translated version uses 填补 (polyfill) with quotation marks, which means it's not a real polyfill. Many Chinese readers may have seen polyfill, but prollyfill is rarely used. I think it could be left alone to reduce misunderstandings, at least people may be able to google the word. I also deleted the duplicated adjective 许多.

![footnote-page-294](https://user-images.githubusercontent.com/47433/135231118-65306c94-4e61-4c44-b286-6432163ae677.png)

**Comment by liunnn1994** [comment]
I agree with that, maybe you can create a PR for this.

**Comment by 0xshubhs** [comment]
Certainly! The issue has been identified in the 1st edition of the book "async & performance" in Chapter 5, Section Title: SIMD. There is a typo where "prolyfill" is used instead of "prollyfill" in a note. The concern is about the term's translation in the Chinese version, where "polyfill" is commonly understood as "填补" (filling in), but "prollyfill" might be unfamiliar. The suggestion is to leave "prollyfill" as is, to encourage readers to look up and discover the term "polyfill." Additionally, a duplicated adjective "许多" has been removed for clarity.
